libs/map/test/depth_point_map_test.cpp

```
@@ -0,0 +1,253 @@
+  EXPECT_TRUE(map.get_points().empty());
+}
+
+TEST(DepthPointMapTest, UpdateWithEmptyCameras) {
+  cuvslam::map::DepthPointMap map;
+  std::vector<cuvslam::map::DepthCameraInfo> empty_cams;
+  map.update(empty_cams);
+  EXPECT_TRUE(map.get_points().empty());
+  EXPECT_FALSE(map.added_last_update());
+}
+
+TEST(DepthPointMapTest, SettingsAreRespected) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)